modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/service/test/SegmentsExperienceLocalServiceTest.java

```diff
@@ -837,6 +879,8 @@ public void testUpdateSegmentsExperienceWithSegmentsExperimentInStatusRunning()
 			RandomTestUtil.randomBoolean());
 	}
 
+	private static final int _PRIORITY_DEFAULT = 0;
```

Comment:
As Priority Default is 0, maybe it is more readable use the numbers directly, because now this number doesn't affect the results, right?

Reply:
Ok, will change it

Reply:
Done! Thanks!